README.md

```diff
@@ -183,6 +195,12 @@ To override the **namespaceMapping** of Avro protocols under the `com.example.pa
                     <to>com.example.packagenamenew.subchange</to>
                 </mapping>
             </namespaceMapping>
+            <fileIncludes>
+                <fileInclude>
+                    <text>subdir/**</text>
```

Comment:
I find `<text>` a bit confusing, how about `<path>`? In your enum you also call it `STRING`, so this is even more confusing.

Reply:
Done